gemrb/plugins/SDLAudio/SDLAudio.cpp

```diff
@@ -31,15 +31,14 @@
 
 using namespace GemRB;
 
-static void SetChannelPosition(const Point &listenerPos, const Point &p, int channel)
+static void SetChannelPosition(const Point &listenerPos, const Point &p, int channel, float rolloff)
 {
 	int16_t angle = AngleFromPoints(listenerPos, p) * 180 / M_PI - 90;
 	if (angle < 0) {
 		angle += 360;
 	}
-	
-	constexpr float AUDIO_DISTANCE_ROLLOFF_MOD (1.3 * 1.3); // squared to save a sqrt
-	uint8_t distance = std::min<uint8_t>((SquaredDistance(listenerPos, p) / AUDIO_DISTANCE_ROLLOFF_MOD), 255u);
+
+	uint8_t distance = std::min(static_cast<int32_t>(std::sqrt(SquaredDistance(listenerPos, p)) / rolloff), 255);
```

Comment:
`std::sqrt(SquaredDistance(X)) == Distance(X)` to a close enough degree.
So you can just use `Distance(listenerPos, p) / rolloff` now that you've started removing the squaring.